Browser option is now supported

okieselbach opened the pull request:
I've tested it several time in different situations and tenants. Since the change of the approved app list which now includes Microsoft Intune Managed Browser the Browser option is supported. I'll give an example, if we want to block browser access from native browsers like Safari or Chrome to enforce the users to use the managed browser we need to build a CA policy for approved app list and check both Browser and Mobile apps and desktop clients. Then this scenario is covered. If we leave browser unchecked, then Safari and Chrome are still working and we do not get the message: "You must use the Intune Managed Browser application before you can access this resource."

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Jim-Parker commented:
@bwren Brian, can you please review this change?

bwren commented:
#sign-off

okieselbach committed:
Browser option is now supported

I've tested it several time in different situations and tenants. Since the change of the approved app list which now includes Microsoft Intune Managed Browser the Browser option is supported. I'll give an example, if we want to block browser access from native browsers like Safari or Chrome to enforce the users to use the managed browser we need to build a CA policy for approved app list and check both Browser and Mobile apps and desktop clients. Then this scenario is covered. If we leave browser unchecked, then Safari and Chrome are still working and we do not get the message: "You must use the Intune Managed Browser application before you can access this resource."
